WordPress/gutenberg
# BorderBoxControl: migrate tests to TypeScript, remove `act()` call

## What?


Part of #35744 

Refactor `BorderBoxControl`'s tests to TypeScript (the rest of the component is already typed)

Remove unnecessary `act()` call in the process

## Why?


Part of the TypeScript migration that we're doing in the `@wordpress/components` package, see #35744 for more info

## How?


## Testing Instructions





No runtime changes. Make sure that the project compiles, and that tests make sense
